cmd/query/app/fixture/index.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
     <meta charset="UTF-8">
+    <base href="/" data-inject-target="BASE_URL"/>
```

Comment:
@tiffon this models the compiled `index.html` from UI assets.

Q: is it possible that babel would reorder the `href` and `data-inject-target` attributes? Because if it does, the search&replace in the query service won't work

Reply:
@yurishkuro It should not change the order of the attributes.